docs/automated-builds.md

```diff
@@ -0,0 +1,27 @@
+# Automated Builds
+
+HNC uses [Google Cloud Build](https://cloud.google.com/build) for building some artifacts.
+Currently only docker builds are automated for every push to the main branch.
+
+## Trigger Setup
```

Comment:
Can you add the name of the GCP project as well as who owns it, so we can get permission to add new people in the future if needed?

Thanks!

Reply:
done